packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -277,6 +277,47 @@ correct log retention period (never expire, by default).
 *Further note* that, if the log group already exists and the `logRetention` is not set, the custom resource will reset
 the log retention to never expire even if it was configured with a different value.
 
+### FileSystem Access
+
+You can configure a function to mount an Amazon Elastic File System (Amazon EFS) file system to a local directory. 
+
+The following sample enables the lambda function to mount the filesystem from Amazon EFS via the `accessPoint` to 
```

Comment:
I think we need to add a few words that explain what are EFS access points, and perhaps a reference to learn more.